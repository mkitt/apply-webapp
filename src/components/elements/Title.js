// @flow
import glamorous from 'glamorous'
import { COLORS } from '../../constants/theme'

export default glamorous.h1({
  fontSize: 48,
  fontWeight: 200,
  color: COLORS.GREY_8,
})
