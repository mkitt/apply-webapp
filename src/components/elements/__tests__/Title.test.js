import React from 'react'
import renderer from 'react-test-renderer'
import Title from '../Title'

describe('<Title />', () => {
  it('shows the correct snapshot tree for the <Title> element', () => {
    const component = <Title>The Title</Title>
    const tree = renderer.create(component).toJSON()
    expect(tree).toMatchSnapshot()
  })

  it('shows the correct snapshot tree for the <Title> element with options', () => {
    const component = (
      <Title color="purple" fontSize={64} mt={8}>
        The Title with options
      </Title>
    )
    const tree = renderer.create(component).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
