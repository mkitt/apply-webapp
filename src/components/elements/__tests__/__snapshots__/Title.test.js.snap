// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Title /> shows the correct snapshot tree for the <Title> element 1`] = `
.css-0,
[data-css-0] {
  font-size: 48px;
  font-weight: 200;
  color: #808080;
}

<h1
  className="css-0"
>
  The Title
</h1>
`;

exports[`<Title /> shows the correct snapshot tree for the <Title> element with options 1`] = `
.css-0,
[data-css-0] {
  font-size: 64px;
  font-weight: 200;
  color: purple;
  margin-top: 512px;
}

<h1
  className="css-0"
>
  The Title with options
</h1>
`;
